core/vm/runtime/runtime_test.go

```diff
@@ -334,11 +334,15 @@ func TestBlockhash(t *testing.T) {
 	if zero.BitLen() != 0 {
 		t.Fatalf("expected zeroes, got %x", ret[0:32])
 	}
-	if first.Uint64() != 999 {
-		t.Fatalf("second block should be 999, got %d (%x)", first, ret[32:64])
-	}
-	if last.Uint64() != 0 {
-		t.Fatalf("last block should be 0, got %d (%x)", last, ret[64:96])
```

Comment:
Can we keep these lines, restoring them to the [original geth test](https://github.com/scroll-tech/go-ethereum/pull/196/files#diff-7951530e74682680c7001b1ff9a86b27b267d22b7eca6f798b38b0c6fa0f0e18)?

Reply:
Are these lines 339-340 and 344-345

Reply:
It's hard to mock the BLOCKHASH this time because we are hashing inside of the opcode execution, so I'm not sure what combination of `chainId` and `block_number` will produce the original geth test results.